apps/backend/scripts/build.mjs

```diff
@@ -0,0 +1,15 @@
+import { build } from 'esbuild'
+
+async function main() {
+  await build({
+    bundle: true,
+    minify: true,
+    platform: 'node',
+    outdir: 'dist',
+    entryPoints: {
+      index: 'src/index.ts',
+    }
+  })
+}
+
+main()
```

Comment:
Why does this need to be a module? From my limited understanding of modules vs scripts, a small program like this counts as a script and would be a `.js` file.

Reply:
No, that's not what the distinction between `mjs` and `cjs` files are. `mjs` files use the new es6 syntax for importing, `cjs` is the old CommonJS system.

```ts
// esm
import path from 'path'

// CommonJS
const path = require('path')
```

A plain `js` file is assumed to be CommonJS unless `"type": "module"` is set in the closest `package.json`

The reason I'm using `mjs` is because there's no `"type": "module"` in the `package.json` and I don't like CommonJS because it's cringe!

Reply:
Makes sense, thx for the explanation